backend/entities/Project.cs

```diff
@@ -1,11 +1,15 @@
 using System;
 using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text.Json.Serialization;
+using Newtonsoft.Json;
 
 namespace Pims.Dal.Entities
 {
     /// <summary>
     /// Project class, provides an entity for the datamodel to manage projects.
     /// </summary>
+    [JsonObject(MemberSerialization.OptIn)]
```

Comment:
Could also have marked all of the DB properties as JsonIgnore, but I think that it makes more sense to keep the property attributes grouped like this.